academy/ibc/ibc-token-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "IBC Token Lifecyle"
+order: 99
+description: Inter-chain token transfers
+tag: fast-track
+---
+
+# Lifecycle of an Inter-chain token transfer
+
+One of topics that tends to confuse newcomers to blockchain technology and crypto assets is exact nature of coins and tokens. The terminology implies the existence of physical or logical objects. Newcomers may wonder where there objects are in physical space. 
```

Comment:
If this is the hook you want to start off with I would recommend making your arguments more explicit:
- Why does the terminology imply the existence of a physical or logical object?
- What is understood as a logical object?
- Will newcomers really wonder where objects are?
I would in general recommend thinking again about the target groups and rechecking the best way is to introduce this topic. Maybe starting off with IBC and how it allows for cross-chain token transfers might be an approach that fits more seamlessly into the overall content for the IBC module.